src/AI/CoreLogic.pm

```diff
@@ -2992,7 +2992,12 @@ sub processAutoAttack {
 	  && timeOut($timeout{ai_attack_auto})
 	  # If !teleportAuto_search, then searchMonsters >= teleportAuto_search will be true - no need for first condition?
 	  && (!$config{teleportAuto_search} || $ai_v{temp}{searchMonsters} >= $config{teleportAuto_search})
-	  && (!$config{attackAuto_notInTown} || !$field->isCity)) {
+	  && (!$config{attackAuto_notInTown} || !$field->isCity)
+	  && ($config{attackAuto_inLockOnly} <= 1 || $field->baseName eq $config{'lockMap'})
+	  && (!$config{attackAuto_notWhile_storageAuto} || !AI::inQueue("storageAuto"))
+	  && (!$config{attackAuto_notWhile_buyAuto} || !AI::inQueue("buyAuto"))
+	  && (!$config{attackAuto_notWhile_sellAuto} || !AI::inQueue("sellAuto"))
```

Comment:
Why do we need new parameters?
i think we can use:
```
	  && (!AI::inQueue("storageAuto"))
	  && (!AI::inQueue("buyAuto"))
	  && (!AI::inQueue("sellAuto"))
```

Reply:
Would work the same, I only thought that maybe some poeple would like to have openkore attack mobs in the route to autostorage/autobuy/autosell

Reply:
Would work the same, I only thought that maybe some poeple would like to have openkore attack mobs in the route to autostorage/autobuy/autosell

Reply:
@alisonrag what do you think?

Reply:
Adding the keys makes it so whoever does not have them added won't have its openkore change behaviour, small upside to adding them I guess.